tests/test_select.py

```diff
@@ -134,6 +142,20 @@ def test_select_position_regular_xy(regular_xy_dataset):
     npt.assert_approx_equal(ds["air"][-1], 279.19), "Temperature is incorrect"
 
 
+def test_select_position_projected_xy(projected_xy_dataset):
+    from xpublish_edr.geometry.common import project_geometry
+
+    point = Point((64.59063409, 66.66454929))
+    projected_point = project_geometry(projected_xy_dataset, "EPSG:4326", point)
+    npt.assert_approx_equal(projected_point.x, 18.045), "Longitude is incorrect"
+    npt.assert_approx_equal(projected_point.y, 21.725), "Latitude is incorrect"
+
+    ds = select_by_position(projected_xy_dataset, projected_point)
+    npt.assert_approx_equal(ds.rlon.values, 18.045), "Longitude is incorrect"
+    npt.assert_approx_equal(ds.rlat.values, 21.725), "Latitude is incorrect"
+    npt.assert_approx_equal(ds.temp.values, 0.89959461), "Temperature is incorrect"
```

Comment:
I dont understand how this fails. The exact code sel query works in a notebook, but in the test context the temperature value is not deterministic, which makes no sense because the coords are correct and it is using nearest neighbor

Reply:
Oh the dataset is generated on load **facepalm**